### DISCUSSION TITLE
How to decrease font size on Visual Studio Code, its taking up my entire screen.

### DISCUSSION THREAD

**Discussion by Sharonesep** [community/community#146811] [Actions] [answered]
### Select Topic Area

Question

### Body

How to decrease font size on Visual Studio Code
[VSC.zip](https://github.com/user-attachments/files/18094615/VSC.zip)

**Comment by MindfulLearner** [comment, marked as answer]
try ctrl + + to size up
or ctrl + - to size down

**Reply by Sharonesep** [reply]
Thank you very much !!

**Reply by MindfulLearner** [reply]
If you found it useful, could you mark this as an answer.

**Reply by MindfulLearner** [reply]
Happy to help anyway thank you!
